packages/test/test-end-to-end-tests/src/test/lessBatches.spec.ts

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { IContainer } from "@fluidframework/container-definitions";
+import { SharedMap } from "@fluidframework/map";
+import { IDocumentMessage } from "@fluidframework/protocol-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import {
+	ChannelFactoryRegistry,
+	DataObjectFactoryType,
+	ITestContainerConfig,
+	ITestFluidObject,
+	ITestObjectProvider,
+	waitForContainerConnection,
+} from "@fluidframework/test-utils";
+import { describeNoCompat } from "@fluidframework/test-version-utils";
+
+describeNoCompat("Less batches", (getTestObjectProvider) => {
```

Comment:
The only way we can add this mode if we
- add a test (equivalent that Navin shared earlier) that demonstrates that code (with timer or not) fails if we inject processing of new incoming batch while having pending outbound batch.
- Fix that problem

It can be separate or same PR, but we have to fix that problem in general. And when using timers, this problem will get workse.